src/utils/api/calls.ts

```diff
@@ -63,6 +73,15 @@ interface PatientsPropsResponse extends BaseApiResponse {
 interface PatientPropsResponse extends BaseApiResponse {
   response: PatientProps
 }
+
+interface FeedbackEditPropsResponse extends BaseApiResponse {
+  response: []
+} 
+
+interface FeedbackPropsResponse extends BaseApiResponse {
+  response: FeedbackProps
+} 
+
```

Comment:
I have a feeling this method is intended to return an array, so I have switched the responses in the refactor branch.